src/components/akoma-ntoso/akoma-ntoso.scss

```diff
@@ -1,7 +1,7 @@
 @import url("https://fonts.googleapis.com/css?family=PT+Serif:400,400i,700,700i");
 
-@import "~indigo-web/scss/mixins";
-@import "~indigo-web/scss/variables";
+@import "node_modules/indigo-web/scss/mixins";
+@import "node_modules/indigo-web/scss/variables";
```

Comment:
when doing an npm link, on tausi, the django sass compiler seemed to be able to read the syntax of `node_modules` instead using tilde

Reply:
This works purely because the `node_modules` is in the current directory when the django command is run. 

Try dropping the tilde, and I'll send you a config change for Tausi.

Reply:
unfortunately if we do this then run `yarn build`, this stencil project will complain that it cannot find the package
```
[ ERROR ]  sass error: src/components/akoma-ntoso/akoma-ntoso.scss:3:9
           Can't find stylesheet to import.

      L3:  @import "indigo-web/scss/mixins";
      L4:  @import "indigo-web/scss/variables";
```

Reply:
one other thing to note is that `node_modules` is included in the contents of the package installed
![included-web-components](https://user-images.githubusercontent.com/15995210/145551418-26ba2015-bf68-40ab-92fd-3081d67c3974.png)
so my theory is that its to use `node_modules` as the reference